lblod/app-lokaal-mandatenbeheer
# verplichte raden

## Description

Automatically create mandatory council organs and their mandates. 

Ready for review but waiting for tim to reply to some business questions:
- only for next period or also the current one?
- did we pick the right adviesraad or should it be the local one, not the commune one?

## How to test

Go to the next government period and see the councils there

![image](https://github.com/user-attachments/assets/ec04dc7b-5989-4c9c-bcd4-c772302e0df1)
